### DISCUSSION TITLE
CONTRACT INTERACTION LESSON ONE

### DISCUSSION THREAD

**Discussion by TSHEPO-CLOUD** [Cyfrin/foundry-full-course-cu#389] [Q&A] [answered]
Morning am not sure if am doing anything wrong interacting with a contract on etherscan, as i get this `Transaction decoding is not available for chainId 11155111` error please assist

**Comment by Equious** [comment, marked as answer]
I don't think you were doing anything wrong. Transaction decoding simply doesn't seem to be supported on Sepolia. Just tested it myself to the same effect.
